tools/testnet/.gitignore

```diff
@@ -0,0 +1 @@
+*.retry
```

Comment:
That needs to be included. It only contains the information necessary to access the actual state stored on s3.